src/tools/genpolicy/genpolicy-settings.json

```
@@ -315,7 +322,8 @@
         "oci_version": "1.1.0"
     },
     "cluster_config": {
-        "pause_container_image": "mcr.microsoft.com/oss/kubernetes/pause:3.6"
+        "pause_container_image": "mcr.microsoft.com/oss/kubernetes/pause:3.6",
+        "is_nydus": false
```

As discussed, to rename this
```suggestion
        "guest_pull": false
```